packages/browser/src/client/tester/context.ts

```diff
@@ -29,7 +29,8 @@ function triggerCommand<T>(command: string, ...args: any[]) {
   return rpc().triggerCommand<T>(contextId, command, filepath(), args)
 }
 
-export function createUserEvent(__tl_user_event__?: TestingLibraryUserEvent): UserEvent {
+export function createUserEvent(__tl_user_event_base__?: TestingLibraryUserEvent): UserEvent {
+  let __tl_user_event__ = __tl_user_event_base__?.setup({})
```

Comment:
I don't understand why this is needed; the first argument is supposed to be called with setup() - see the `setup` method on L40

Reply:
Not sure if this what you're asking, but we need to pass around the original `userEvent` since this happens:

```ts
import { userEvent } from "@testing-library/user-event"

const userEvent1 = userEvent.setup();
userEvent1.keyboard("{Alt>}")

// userEvent1 and userEvent2 share the same keyboard state 
const userEvent2 = userEvent1.setup();
userEvent2.keyboard("{Tab}") // altKey = true


// need to call original `userEvent.setup()` to have a fresh state
{
  const userEvent1 = userEvent.setup();
  userEvent1.keyboard("{Alt>}")
  
  const userEvent2 = userEvent.setup();
  userEvent2.keyboard("{Tab}") // altKey = false
}
```

For L40, we should probably recurse it like `createUserEvent(__tl_user_event_base__)`.